docs/content/csi_rn_compatibility.md

```diff
@@ -0,0 +1,3 @@
+# Compatibility and requirements
+
+This section specifies the compatibility and requirements of version 1.6.0 of IBM® block storage CSI driver.
```

Comment:
could you add the ticket number to fill this file? we don't want to miss any "TODOs"

Reply:
thanks!

Reply:
@oriyarde can we close this issue? @erantzabari  will be reviewing the documentation later today